bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,355 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<thing:thing-descriptions bindingId="freeboxos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="lcd-brightness" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Screen Brightness</label>
+		<description>Brightness level of the screen in percent</description>
+		<category>DimmableLight</category>
+		<state pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="lcd-orientation">
+		<item-type>Number</item-type>
+		<label>Screen Orientation</label>
+		<description>Screen Orientation in degrees</description>
+		<state pattern="%d °">
+			<options>
+				<option value="0">Horizontal</option>
+				<option value="90">Turned left</option>
+				<option value="180">Reversed</option>
+				<option value="270">Turned right</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="rssi">
```

Comment:
This should be changed to use the abstracted system channel for this (system.signal-strength)... See https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types

You can see an example of this in a PR I am getting reviewed at the moment, see opensprinkler PR that is open at the moment.

Reply:
There is a second channel for the abstracted version (wifi-quality). This one is the raw value in dB.